catalog/plugin/catalog-plugin-querymonitorcommands/src/main/java/ddf/catalog/querymonitorcommands/PrintActiveSearchesCommand.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.querymonitorcommands;
+
+import java.io.PrintStream;
+import java.util.Map;
+import java.util.UUID;
+
+import org.apache.karaf.shell.api.action.Action;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.fusesource.jansi.Ansi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.querymonitor.ActiveSearch;
+import ddf.catalog.querymonitor.QueryMonitorPlugin;
+
+@Command(scope = "catalog", name = "printactivesearches", description = "Prints a summary of all ActiveSearches currently in the QueryMonitor")
+@Service
+public class PrintActiveSearchesCommand implements Action {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PrintActiveSearchesCommand.class);
+
+    @Reference
+    private QueryMonitorPlugin queryMonitor = null;
+
+    private static final String FORMAT_STRING = ActiveSearch.FORMAT_STRING;
+
+    PrintStream console;
+
+    PrintStream getPrintStream() {
+        return System.out;
+    }
+
+    public PrintActiveSearchesCommand(QueryMonitorPlugin q) {
+        queryMonitor = q;
+        console = getPrintStream();
+    }
+
+    public PrintActiveSearchesCommand() {
+        console = getPrintStream();
+    }
+
+    /**
+     * Method to implement {@link Action}. Called when the catalog:printacticesearches command is called by the shell.
+     * Prints out a formatted table representing a {@link QueryMonitorPlugin}'s {@link ActiveSearch} {@link Map}.
+     *
+     * @return always returns {@code null}
+     */
+
+    @Override
+    public Object execute() throws Exception {
+
+        try {
+            printActiveSearchesToConsole();
+        } catch (Exception e) {
+            LOGGER.debug("Exception encountered in doExecute of PrintActiveSearchesCommand.java. {}",
```

Comment:
If you log an exception like this, you won't get the stack trace. You can just pass the exception as the second argument and remove the `{}` in the message. Also, it seems like `warn` or `error` would be more appropriate.